src/lib/constants.ts

```diff
@@ -1,7 +1,9 @@
-export const ACTOR = process.env.ACTOR ?? 'eosio.faucet';
-export const ACCOUNT = process.env.ACCOUNT ?? ACTOR;
+import { PUBLIC_ACTOR } from "$env/static/public";
+
+export const ACTOR = PUBLIC_ACTOR;
+export const ACCOUNT = 'eosio.faucet';
 export const EVM = 'eosio.evm';
-export const FAUCET = 'eosio.faucet';
+export const FAUCET = ACCOUNT;
 export const PERMISSION = process.env.PERMISSION ?? 'active';
 export const BROADCAST = Boolean(process.env.BROADCAST ?? 'true');
```

Comment:
remove `process.env.` You may set to literal string or you can move them to the `.env` file and import them along with `PUBLIC_ACTOR` in line 1